lib/rubocop/cop/rspec_rails/minitest_assertions.rb

```diff
@@ -315,6 +315,27 @@ def assertion
           end
         end
 
+        # :nodoc:
+        class ResponseAssertion < BasicAssertion
+          MATCHERS = %i[
+            assert_response
+          ].freeze
+
+          # @!method self.minitest_assertion(node)
+          def_node_matcher 'self.minitest_assertion', <<~PATTERN # rubocop:disable InternalAffairs/NodeMatcherDirective
+            (send nil? {:assert_response} $_ $_?)
+          PATTERN
+
+          def self.match(expected, failure_message)
+            response = Struct.new(:source).new('response')
+            new(expected, response, failure_message.first)
```

Comment:
just noting we're using `Struct` here because the "actual" for this assertion is the test-local `@response` property - I think this is a good solution especially as `BasicAssertion` immediately calls `source` and stores that value rather than holding any further reference to the `actual` parameter, though I wouldn't be surprised if there is a Rubocop helper for these situations that we could use instead